components/builder-sessionsrv/src/server/handlers.rs

```diff
@@ -92,3 +156,27 @@ pub fn session_get(req: &mut Envelope,
     }
     Ok(())
 }
+
+// Determine permissions and toggle feature flags on for the given Sesssion
+fn set_features(state: &ServerState, session: &mut proto::Session) -> Result<()> {
```

Comment:
That's right, in the session create/session get handlers we call `set_features` on the session before sending it as a response to the transaction